### DISCUSSION TITLE
Prevent autofocus on opening sheet

### DISCUSSION THREAD

**Discussion by Maniktherana** [huntabyte/shadcn-svelte#1002] [Help]
So I have a sheet component which has a form in it. whenenver i open it it focuses on the input element by default. how can i prevent this? Edit: setting autofocus={false} on the input doesn't work

**Comment by devjume** [comment]
Use `openFocus` to set the focus to be on `content` div via its `id`.  
This focuses on the parent div and not input element.

```
<Sheet.Root openFocus={"#content-div"}>
  <Sheet.Content id="content-div">
    <input type="text"/>
  <Sheet.Content/>
</Sheet.Root>
```
See my comment for more details on [melt-ui #1007](https://github.com/melt-ui/melt-ui/issues/1007#issuecomment-2094284258) 
Also discussed on [issue #427](https://github.com/huntabyte/bits-ui/issues/427)

**Comment by ghoshsanjoy78** [comment]
This causes an outline on the sheet component (since the content div is auto focused). Not ideal either. Any other solutions ?
